storage/engine.go

```diff
@@ -222,6 +223,10 @@ func (e *Engine) Close() error {
 		retErr = multierror.Append(retErr, fmt.Errorf("error closing TSDB store: %w", err))
 	}
 
+	if err := e.pointsWriter.Close(); err != nil {
+		retErr = multierror.Append(retErr, fmt.Errorf("error closing points writer: %w", err))
```

Comment:
Looks like this is the only package in this repository which imports github.com/hashicorp/go-multierror, but two other packages import go.uber.org/multierr. Do you mind switching this to go.uber.org/multierr?

Reply:
done